Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -2432,17 +2506,10 @@ def panorama_get_url_category(url_cmd: str, url: str, target: Optional[str] = No
     if 'Failed to query the cloud' in result:
         raise Exception('Failed to query the cloud. Please check your URL Filtering license.')
 
-    if url_cmd == 'url-info-host':
-        # The result in this case looks like so: "Ancestors info:\nBM:\nURL.com,1,5,search-engines,, {some more info
-        # here...}" - The 4th element is the url category.
-        category = result.split(',')[3]
-    else:
-        result = result.splitlines()[1]
-        if url_cmd == 'url':
-            category = result.split(' ')[1]
-        else:  # url-info-cloud
-            category = result.split(',')[3]
-    return category
+    for url_category in PAN_DB_URL_FILTERING_CATEGORIES:
+        if url_category in result:
+            categories.append(url_category)
+    return categories
```

Comment:
> nicely done, make sure you test all the commands that are being affected that nothing breaks.
> 
> just to make sure, the context stays the same only that now it will be a list, right?

The context be a list of categories, each one of them will have a list of URLs which belongs to the category. For example: 
<img width="235" alt="image" src="https://user-images.githubusercontent.com/38375556/178251545-f2a9e68e-abd4-4690-a0ee-67f354c79115.png">